tests/batch/test_get_list_batches.py

```diff
@@ -0,0 +1,485 @@
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--start',
+        start,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+
+def test_get_list_batches_with_non_existing_start():
+    """
+    Case: get a list of batches by non-existing batch identifier starting from.
+    Expect: list of batches not found error message.
+    """
+    start = 'fa2d1a209ad04fd2ad7fb5183976e647cc47b4c08e2e578097afc2566a0284e7' \
+            '60eb3f2ff8f72f290765211d4da3341f23091cc7a16805025a17c04a90818a45'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--start',
+        start,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': 'List of batches not found.',
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_batches_with_reverse():
+    """
+    Case: get a list of batches by reverse.
+    Expect: reversed list of batches is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+        '--reverse',
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+
+def test_get_list_batches_by_head():
+    """
+    Case: get a list of batches by block identifier.
+    Expect: batches from specified block are returned.
+    """
+    head = '56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a' \
+           '40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c551'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--head',
+        head,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+
+def test_get_list_batches_by_non_existing_head():
+    """
+    Case: get a list of batches by non-existing block identifier.
+    Expect: list of batches not found error message.
+    """
+    head = '56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a' \
+           '40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c552'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--head',
+        head,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': 'List of batches not found.',
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+@pytest.mark.parametrize('command_flag', ('--start', '--head'))
```

Comment:
```suggestion
@pytest.mark.parametrize('command_flag', ('--ids', '--start', '--head'))
```
Add argument `ids` to this test case and delete test `test_get_list_batches_with_invalid_ids `.

Reply:
Done.